mdflynn/game-sleuth
# Bug/search results

## Is this a fix or a feature?
This is a fix

## What is the change?
This adds the ability to hit "Enter" to have search results returned

## What does it fix?
Corrects issue where apicall was not including the min/max values

## Where should the reviewer start?
Lines 17 & 58-62 in SearchForm.tsx

## How should this be tested?
Run `npm start` and navigate to localhost: 3000 to see current site functionality

## Relevant Issues
#50 #51
